docker/cassandra/install-cassandra

```diff
@@ -42,6 +62,40 @@ mkdir /etc/cassandra/logback.xml.d
     curl -SLO "https://github.com/instaclustr/cassandra-exporter/releases/download/v0.9.6/cassandra-exporter-agent-0.9.6.jar" &&
     ln -s cassandra-exporter-agent-0.9.6.jar cassandra-exporter-agent.jar)
 
+# install Instaclustr LDAP and Kerberos authn/authz plugins
+(
+    cd "/usr/share/cassandra/lib"
+
+    # TODO: switch remaining Maven Central URLs to GitHub.
+    case ${cassandra_version} in
+    3.11.*)
+        curl -SLO "https://github.com/instaclustr/cassandra-ldap/releases/download/3.11.4.4/cassandra-ldap-3.11.4.jar"
+        ;;
+    3.0.*)
+        curl -SLO "https://search.maven.org/remotecontent?filepath=com/instaclustr/cassandra-ldap/3.0.18.0/cassandra-ldap-3.0.18.0.jar"
+        ;;
+    2.2.*)
```

Comment:
Is it worth doing 2.2? Given the rest of the operator doesnt support it?

Reply:
Added this before I tested 2.2. Then I realized that does some weird things on install and even fails to start, hence the test and comments above (line 12+).

Happy to remove this. Your call.